index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Queen Elizabeth</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Queen Elizabeth II</h1>
+        <p id="text-title">The Queen of England from 1952 to 2022</p>
+        <div id="img-div">
+            <img src="https://cdn.pixabay.com/photo/2016/02/16/15/39/queen-1203465__340.jpg" alt="Queen Elizabeth" id="image">
```

Comment:
- [ ] Ouch! this is close but please make sure you provide an explanation for each suggestion you make It's important to give a clear reason why you are making this suggestion, as it will help the developer understand the importance of adding alternative text to the image. ✔
- You could for example say: "The reason why it's important to provide alt text is that it helps people with visual impairments to understand the content of the image. Alternative text is read aloud by screen readers and other assistive technologies, allowing people who can't see the image to understand what it depicts." 😉